Source/CombatExtended/CombatExtended/StatWorkers/StatWorker_ArmorPartial.cs

```diff
@@ -19,30 +19,41 @@ public override string GetExplanationFinalizePart(StatRequest req, ToStringNumbe
                 {
                     if (req.Thing.def.HasModExtension<PartialArmorExt>())
                     {
-                        var result = "CE_StatWorker_ArmorGeneral".Translate() + finalVal.ToString() + " \n \n" + "CE_StatWorker_ArmorSpecific".Translate();
+                        var result = "";
+
+                        if ((this.stat?.parts ?? null) != null)

```

Comment:
No need for `?? null` when comparing to `null` right after.
```suggestion
                        if (this.stat?.parts != null)

```